libs/permissions/api/src/lib/permission.decorator.ts

```diff
@@ -1,7 +1,14 @@
-import {SetMetadata} from '@nestjs/common'
+import {applyDecorators} from '@nestjs/common'
+import {OneOf} from '@wepublish/authentication/api'
+import {PermissionsGuard} from './permission.guard'
 import {Permission} from './permissions'
+import {AddMetadata} from '@wepublish/nest-modules'
 
 export const PERMISSIONS_METADATA_KEY = 'permissions'
 
+/**
+ * Causes the method/class to require that the current user has the permission to do the action.
```

Comment:
@Itrulia 
If remember correctly these annotation influencing guard behaviour cannot be put on any class/method but just those related to endpoints like controller or resolvers, right?

Reply:
They work now because I changed it to use getAllAndMerge (there's also getAllAndOverride) instead of get. I pass in the class and handler like so:

```
context.getHandler(),
context.getClass()
```